apps/auth-proxy/routes/[...auth].ts

```diff
@@ -11,6 +11,12 @@ export default eventHandler((event) =>
       Github({
         clientId: process.env.GITHUB_ID,
         clientSecret: process.env.GITHUB_SECRET,
+        profile: (p) => ({
```

Comment:
i don't think this is necessary, since the proxy shouldn't touch this thing anyway but put it in here to be sure